web/views/datasets.py

```diff
@@ -58,7 +59,14 @@ def get_context_data(self, **kwargs):
         context['can_edit'] = self.request.user.can_edit_dataset(self.object)
         context['can_see_protected'] = self.request.user.has_permission_on_object(f'core.{Permissions.PROTECTED.value}_dataset', self.object)
         context['company_name'] = COMPANY
+
+        exposure_form_names_none = Exposure.objects.filter(dataset=self.object, form_name=None)
+        for exposure in exposure_form_names_none:
+            rems_forms_dict = get_rems_forms(exposure.form_id)
+            exposure.form_name = rems_forms_dict["form/internal-name"]
+            exposure.save()
```

Comment:
Is this to avoid displaying a dataset with an exposure which has no name stored?

I have the feeling this would be better to just display a placeholder character in the template if the form name is None.

We should discuss this on Monday I think